packages/nextjs-kit/src/lib/sortChar.ts

```diff
@@ -0,0 +1,41 @@
+import { DataToSort, ASC, DESC } from '../types';
+
+/**
+ *
+ * @param {Array<Record<string, string | number>>} obj object to sort
+ * @param {string} key to sort object by
+ * @param {string} direction to sort object by
```

Comment:
For the params, we can use the types from our interface instead of declaring the primitives here:

```suggestion
 * @param {DataToSort['data']} sortObj.data - The data sort
 * @param {DataToSort['key']} sortObj.key - Key to sort object by
 * @param {DataToSort['direction']} sortObj.direction - Direction to sort the data
```